src/GraphControl/Control/Grapher.cpp

```diff
@@ -635,4 +644,45 @@ namespace GraphControl
             }
         }
     }
+
+    RandomAccessStreamReference^ Grapher::GetGraphBitmapStream()
+    {
+        RandomAccessStreamReference^ outputStream;
+
+        if (m_renderMain != nullptr && m_graph != nullptr)
+        {
+            if (auto renderer = m_graph->GetRenderer())
+            {
+                std::shared_ptr < MathSolverEngine::Graph::Renderer::IBitmap> BitmapOut;
```

Comment:
This should be from the Graphing namespace

Reply:
Fixed